### ISSUE TITLE
A bunch of stuff to get deleted and left as nothing ever happened again

### ISSUE THREAD

**Issue by TheAirBlow** [Vawlpe/MOLE#48]
# Beginning
You are just trying to make yourself the best programmer and the best guy in the world.
You state working stuff as "broken" by me. Of course.
They all work, but may be done in a shitty way.

The way you just removed everything I and you've said in the #47 is fucking pathetic. It looks ugly and looks that I was the only who wrote something, your deleted messages just disappeared without any trace (cause you are a collaborator?)

## Commit b85d3f7d4fc2e53c594a83c7d604e66e2dd35bdd
1) [Difference](https://github.com/Vawlpe/MOLE/commit/b85d3f7d4fc2e53c594a83c7d604e66e2dd35bdd#diff-50338035ffb3461102e527f09350262e0cde317156f0fad3842645380f399370L30). You just deleted some windows and rearranged them, also deleted styling stuff (Round buttons, windows, etc.)
2) [Difference](https://github.com/Vawlpe/MOLE/commit/b85d3f7d4fc2e53c594a83c7d604e66e2dd35bdd#diff-8d47af42b87ad746d264cba5c498cf6c86960512a94e0e92ed0a4715074acf48L17) and [Difference](src/Mole.Veldrid/VeldridController.cs). Deleted unused method and did some styling stuff (Window title), which contradicts with you've did in [Difference](https://github.com/Vawlpe/MOLE/commit/b85d3f7d4fc2e53c594a83c7d604e66e2dd35bdd#diff-50338035ffb3461102e527f09350262e0cde317156f0fad3842645380f399370L30).
> You state working stuff as "broken" by me. Of course.

More stuff about it - here you only fixed the GFX window made by me. And not fixed, but rewritten to be an editor - which is good. But saying that you fixed `a bunch of stuff @TheAirBlow broke` is not nice at all.

## Commit 099580f13445c54365c117c00262b22e8d94223d
So you deleted everything you did and stated it is broken. It makes me feel so good.
1) [Difference](https://github.com/Vawlpe/MOLE/commit/099580f13445c54365c117c00262b22e8d94223d#diff-569e0f50abac5e23db37292c610a3c904ef24fef1a327d430fd035243d6ac174L1). How it is broken? You moved it into new class you coded in bb745e28e69f63d40bc7966ee8388979b96f6f85, which makes the commit message incorrect cause it doesn't describe the actual commit.
2) [Difference](https://github.com/Vawlpe/MOLE/commit/099580f13445c54365c117c00262b22e8d94223d#diff-569e0f50abac5e23db37292c610a3c904ef24fef1a327d430fd035243d6ac174L1). Just deleted the entire project loading and it's creating. I know that `dev` stuff isn't supposed to work, but it **must** do the basic stuff. If it's undone it should be in `Debugging` menu or there's should be no way of accessing it. First you code the thing, and only after you get it to work. If you delete something, you should replace it.

## Commit bb745e28e69f63d40bc7966ee8388979b96f6f85
Looks so fucking ugly.
1) [Line 13](https://github.com/Vawlpe/MOLE/commit/bb745e28e69f63d40bc7966ee8388979b96f6f85#diff-509e6494b2563a7e4cfd55fe063deaa94b4c7c3078e4b54452280c901222c2f3R13). Saves info and data without any information about it's length which makes loading it impossible.
2) [Line 25](https://github.com/Vawlpe/MOLE/commit/bb745e28e69f63d40bc7966ee8388979b96f6f85#diff-509e6494b2563a7e4cfd55fe063deaa94b4c7c3078e4b54452280c901222c2f3R25) and [Line 31](https://github.com/Vawlpe/MOLE/commit/bb745e28e69f63d40bc7966ee8388979b96f6f85#diff-509e6494b2563a7e4cfd55fe063deaa94b4c7c3078e4b54452280c901222c2f3R31). Just a shitty shorthand for `File.ReadAllBytes()`. Hash check should be performed on project load. Data, information, type are nowhere to be seen.

## Other stuff
You've haven't fixed the bug that I stated in #47 that speeds up GFX decompression because of an expensive `.ToArray()` call - instead you just ignored it and refused to do it, cause it will change the way to access the ROM. It's like "I won't fix that cause I don't want to do it in a better way, I'll keep it slow and shitty instead". If you wanted to do it, you would've already fixed it, as the latest commit was done on November 2nd. **My bad ass personality is not an argument here, stop using it.**

# The End
You can't delete the issue itself, only hide it by closing it, go cry. All your and my responses will be written here so they wouldn't get deleted :)

**Comment by Vawlpe** [comment]
you realize.... deleting issues is very easy...
but thx for the "feedback" dumfuck, i only pushed changes because i don't want them to be lost locally, not because they're final, and i've got a bunch of stuff i'm working on and i don't have time to deal with you, also as for the speed up for decomp stuff... i already committed that lol, i just have yet to push it, anyway, imma delete this shit now, go fuck yourself and never come back.
